pkgs/os-specific/linux/busybox/default.nix

```diff
@@ -159,6 +159,8 @@ stdenv.mkDerivation rec {
 
   doCheck = false; # tries to access the net
 
+  passthru.shellPath = "/bin/ash";
```

Comment:
I think the relevant links are to [`toShellPath`](https://github.com/NixOS/nixpkgs/blob/0fcbda8294f9de3bbfe44ffc9d4c113895efa816/nixos/lib/utils.nix#L112) and [`shellPackage`](https://github.com/NixOS/nixpkgs/blob/0fcbda8294f9de3bbfe44ffc9d4c113895efa816/lib/types.nix#L514); correct me if I'm wrong.